cypress/support/fragments/bulk-edit/bulk-edit-actions.js

```diff
@@ -99,6 +99,12 @@ export default {
     cy.expect(HTML('Replace with').is({ disabled: true }));
   },
 
+  replaceEmail(oldEmailDomain, newEmailDomain) {
+    getBulkEditSelectType().select('Email');
+    cy.get('[class^=textField]').first().type(oldEmailDomain);
```

Comment:
If interactor 'TextField' not working move to const

Reply:
thx, fixed